src/script/script.cpp

```diff
@@ -329,6 +329,15 @@ bool CScript::IsNullAssetTxDataScript() const
             (*this)[0] == OP_RVN_ASSET &&
             (*this)[1] == 0x14);
 }
+
+bool CScript::IsNullGlobalRestrictionAssetTxDataScript() const
+{
+    // 3 OP_RVN ASSETS + atleast 4 characters for the restricted name $ABC
```

Comment:
(see above re 3xOP_RVN_ASSET -- guessing the comments are just misleading me)

Reply:
Fixed